src/api/app/services/rpmlint_log_parser.rb

```diff
@@ -0,0 +1,51 @@
+class RpmlintLogParser
+  attr_reader :errors, :badness, :warnings, :info
+
+  def initialize(content:)
+    @content = content || ''
```

Comment:
```suggestion
    @content = content || []
```

Reply:
We're iterating over it below, so it should be initialized to an enumerable

Reply:
We're iterating over a string below, using [each_line](https://ruby-doc.org/3.1.4/String.html#method-i-each_line).

I would agree with you if we were iterating over an enumerable, which doesn't have the `each_line` method.

Reply:
Alright, it understand new lines so it works... Dismissing